social_django/utils.py

```diff
@@ -24,7 +23,7 @@ def load_strategy(request=None):
 
 
 def load_backend(strategy, name, redirect_uri):
-    Backend = get_backend(BACKENDS, name)
+    Backend = get_backend(settings.AUTHENTICATION_BACKENDS, name)
```

Comment:
This will also load non social-auth backends.

Reply:
@omab Any update on this?  Is @timabbott's approach satisfactory?

Reply:
I'll chime in and say I'd still love a fix for this - I ended up having to work around it in a somewhat gross way.